### DISCUSSION TITLE
Feasibility for option to load embedding model from local machine

### DISCUSSION THREAD

**Discussion by harshakhmk** [milvus-io/milvus#29095] [Q&A and General discussion]
Hi, I am currently exploring various features of Milvus, and was searching if there is an option to load a sentence Transformer model like all-MiniLM-L6 from my PC so that I can directly give path to use this model without contacting hugging face all the time

**Comment by yhmo** [comment]
Milvus only stores and ANN searches for vectors, and doesn't generate vectors. So, it doesn't maintain any embedding models, it doesn't depend on any embedding models.
All the model-related work is in the upper layer application level, and has nothing to do with Milvus..

**Reply by harshakhmk** [reply]
Yes I understand, it is an upper layer, are there any tutorials or documentation for running the Milvus and ML locally
